spec/features/event_disavowal_spec.rb

```diff
@@ -22,14 +22,15 @@
     disavow_last_action_and_reset_password
   end
 
-  scenario 'disavowing a new device sign in' do
-    allow(IdentityConfig.store).to receive(:otp_delivery_blocklist_maxretry).and_return(3)
-    signin(user.email, user.password)
-    Capybara.reset_session!
-    visit root_path
-    signin(user.email, user.password)
-
-    disavow_last_action_and_reset_password
+  unless FeatureManagement.aggregate_new_device_alerts?
```

Comment:
To test configurable features, we should have either a "default" behavior and a `context` for the non-default configuration, or a `context` for each. I wouldn't expect conditional test cases.

For example:

```rb
context 'when aggregated new device alerts is disabled' do
  before do
    allow(IdentityConfig).to receive(: feature_new_device_alert_aggregation).and_return(false)
  end

  # ...
end

context 'when aggregated new device alerts is enabled' do do
  before do
    allow(IdentityConfig).to receive(: feature_new_device_alert_aggregation).and_return(true)
  end

  # ...
end
```

Reply:
I see, so to pass the configuration into the context. I have updated those conditionals.